<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botonPausar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pausar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonReanudar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reanudar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEmpezar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Empezar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPausar_Balancer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>390</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pausar Balancer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReanudar_Balancer">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reanudar Balancer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextColaPedidos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cola de Pedidos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblColaPedidos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextBalanceador">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Balanceador</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBalanceador">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextColaAlisto">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cola de Alisto</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblColaAlisto">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextAlistador1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alistador 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextAlistador2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alistador 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextAlistador3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alistador 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextAlistador4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alistador 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextAlistador5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alistador 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextAlistador6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Alistador 6</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlistador1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlistador2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlistador3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlistador4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlistador5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlistador6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextColaAlistado">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cola Alistado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblColaAlistado">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextEmpacador">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Empacador</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmpacador">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextFabrica1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fabrica A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFabrica1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextFabrica2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fabrica B</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFabrica2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextFabrica3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fabrica C</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFabrica3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextFarbicaComodin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fabrica Comodin</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFabricaComodin">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextPorFcaturar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Por Facturar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPorFacturar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTextFacturador">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Facturador</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFacturador">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArmazon">
    <property name="title">
     <string>Armazon</string>
    </property>
   </widget>
   <addaction name="menuArmazon"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
